### PR TITLE
Support Cloudshark's new API.

### PR DESCRIPTION
**Pull request by kgoodier**
Cloudshark now requires you to create an account and use an API token
to upload files. See https://support.cloudshark.org/api/upload.html
for more info.

The /etc/config/cdns file must now be edited after installation in
order to provide an API token.

Resolves issue https://github.com/Polidea/Cellular-Data-Network-Simulator/issues/7

### CODE REVIEW

**Comment by kkocel** [comment]
LGTM :)

File: www/cgi-bin/dump [hunk 1]
```diff
@@ -95,7 +101,7 @@ start)
 	for i in 1 2 3
 	do
 		push_event $EVENT_ID UPLOAD "Uploading capture for ${PATHS[2]} to CloudShark ($i try)..."
-		JSON_STATUS=`curl --cacert /usr/lib/cdns/cacert.pem -H "Origin: https://appliance.cloudshark.org" -F "capture[upload]=@${DUMP_FILE}.0" https://www.cloudshark.org/captures.json`
+		JSON_STATUS=`curl --cacert /usr/lib/cdns/cacert.pem --upload-file ${DUMP_FILE}.0 https://www.cloudshark.org/api/v1/${CLOUDSHARK_API_TOKEN}/upload`
```
Comment (ayufan) [inline www/cgi-bin/dump]: Don't we need "" around ${DUMP_FILE}.0?

Comment (kgoodier) [inline www/cgi-bin/dump]: Yeah, wouldn't hurt. Will update.

Comment (kgoodier) [inline www/cgi-bin/dump]: Took a look, and nowhere else in that file is $DUMP_FILE surrounded by double quotes. While I agree it would be safer to do that, this change matches existing usage. Quoting it should probably be a separate commit outside the scope of this pull request.

Comment (ayufan) [inline www/cgi-bin/dump]: It was in "capture[upload]=@${DUMP_FILE}.0"

**Comment by ayufan** [comment]
@kgoodier can fix this quotes?

**Comment by kgoodier** [comment]
Updated and added quotes to DUMP_FILE in the curl line.

**Comment by ayufan** [comment]
Thanks!
